packages/contracts-diamond/contracts/assets/ERC721/ERC721DropSinglePhaseFacet.sol

```diff
@@ -41,16 +109,18 @@ contract ERC721DropSinglePhaseFacet is IERC721DropSinglePhase {
 
     /**
      * @notice Check if a given account and its max claim are valid for a specific drop condition.
+     * @param merkleRoot The Merkle root to use for verification.
      * @param account Address of the account to check.
      * @param accountMaxClaim Maximum number of tokens the account can claim.
      * @param proof Array of hashes required to verify the account's eligibility.
      * @return boolean indicating if the proof is valid for the given account and max claim.
      */
     function checkMaxClaimForAccount(
```

Comment:
Remove. Not necessary

Reply:
Add accessor for storage
* getDropConditionMerkleRoot
* getDropconditionUserClaimed